### PR TITLE
Proposal to Change Representation of Zone Equipment

### PR DESCRIPTION
**Pull request by jasondegraw**
The schema presently does not allow for the representation of the presence of delivery-type zone equipment without specifying the type of zone equipment. Workarounds are possible, but all involve adding either additional enumerations or the use of user-defined fields. The proposed change splits the fan-based delivery equipment into two parts and branches on "zone equipment or not" before branching on the equipment type.

### CODE REVIEW

File: proposals/2018/Explicit_Zone_Equipment_Representation.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    - FanBasedDistributionType  
+       - CentralAirDistribution   
+       - ZoneEquipment (New)  
+          - FanCoil  
+          - Convection  
+          - Radiant  
+          - Other  
+        - Other (New)  
+
+Recommended Option A: This reflects what was requested originally with small modifications to make it more intuitive where CentralAirDistribution and ZoneEquipment are children of DeliveryType not FanBasedDistributionType. Pro: Similar to request. Con: ZoneEquipment has children (Convection, Radiant, etc) which might be limiting to use cases.
```
Comment (nllong) [inline proposals/2018/Explicit_Zone_Equipment_Representation.md]: This change adds another level to the DeliveryType. The DeliveryType presently includes FanBased, Convection, Radiant, and Other. Moving `CentralAirDistribution` to the DeliveryType promotes the idea of CentralAirDistribution outside the scope of fan-based delivery when central air distribution is a fan-based delivery.

File: proposals/2018/Explicit_Zone_Equipment_Representation.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  - ZoneEquipment (New)  
+     - FanBased  
+       - FanBasedDistributionType   
+            - FanCoil  
+     - Convection  
+     - Radiant  
+     - Other  
+  - Other (New)  
+
+Recommended Option B: This is furthest from what was originally requested but would meet the goals of the 211 use case cleanly. Pro: Structure is intuitive. Con: Most changes.
```
Comment (nllong) [inline proposals/2018/Explicit_Zone_Equipment_Representation.md]: I like this option and how it breaks up the zoning type and delivery type. I am not sure how the two items become linked.

File: proposals/2018/Explicit_Zone_Equipment_Representation.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+  * Hydronic to zone equipment (e.g. fan coil units, packaged terminal units or radiators)
+  * Refrigerant to zone equipment (e.g. fan coil units, packaged terminal units or radiators)
+
+All of the other checkboxes can be represented without ambiguity. Non-breaking workarounds are available (e.g. add an enumeration value), but this means that the representations of the various systems are inconsistent. A breaking change that makes the representations consistent is preferable, and a making the breaking change now instead of a breaking change later will reduce the impact of the change.
+
+## Implementation ##
+There are three options for implementation of the Zone Equipment representation in the Delivery section. 
+
+Originally Proposed: This is what we understood to be proposed originally. It is specified with no missing relevent parent/child relationships. Pro: Reflects request. Con: Organization of information is a bit confusing with having the ZoneEquipment as a child of the  FanBasedDistributionType.
```
Comment (nllong) [inline proposals/2018/Explicit_Zone_Equipment_Representation.md]: I still recommend using the original proposal. @supriyagoel can you please provide input on what you think is best?

File: proposals/2018/Explicit_Zone_Equipment_Representation.md (hunk 2 above)
Comment (supriyagoel) [reply proposals/2018/Explicit_Zone_Equipment_Representation.md]: I like option B as well. 
Option A would be slightly incorrect as radiant systems shouldn't be categorized under 'FanBased'.
Option B is very clean and maps to 211. @nllong I couldn't understand what you mean you mean by 'zoning type'

Comment (nllong) [reply proposals/2018/Explicit_Zone_Equipment_Representation.md]: 'zoning type' was the new 'DeliveryZoningType' element. If we are in agreement, then I think we should go with option B. Option B work for you @supriyagoel, @jasondegraw, and @aqwolfe?

Comment (aqwolfe) [reply proposals/2018/Explicit_Zone_Equipment_Representation.md]: @nllong yes, prefer option B

Comment (jasondegraw) [reply proposals/2018/Explicit_Zone_Equipment_Representation.md]: @nllong For the record, I don't think my original proposal was understood. I was not suggesting that radiant systems etc. be included as a child of fan-based equipment.

But setting that aside and focusing on the two current options: Option A. Sorry to be the lone voice of dissent, but Option A conforms much better to the way that other 211 data is best represented, while Option B is too flat and allows for easy specification of contradictory data. I also don't understand the stated con for A:

> ZoneEquipment has children (Convection, Radiant, etc) which might be limiting to use cases.

In what sense? Maybe if I understood that better I'd understand where the preference for B is coming from, but from my point of view Option A is the way to go.

**Pull request merged**